storage/storage.hpp

```diff
@@ -246,16 +246,18 @@ class Storage
   /// returns in param all countries ids. It's content of map downloader list by default.
   void GetChildren(TCountryId const & parent, TCountriesVec & childrenId) const;
 
-  /// \brief Fills localChildren with children of parent.
-  /// The result of the method is composed in a special way because of design requirements.
-  /// If a direct child (of parent) contains two or more downloaded mwms the direct child id will be added to result.
-  /// If a direct child (of parent) contains one downloaded mwm the mwm id will be added to result.
-  /// If there's no downloaded mwms contained by a direct child the direct child id will not be added to result.
-  /// \param parent is a parent acoording to countries.txt.
-  /// \note. This method puts to localChildren only real maps which have been written in coutries.txt. It means
-  /// the method does not put to localChildren neither custom maps generated by user
+  /// \brief Fills |downloadedChildren| and |availChildren| with children of parent.
+  /// If a direct child of |parent| contains at least one downloaded mwm
+  /// the mwm id of the child will be added to |downloadedChildren|.
+  /// If not, the mwm id the child will not be added to |availChildren|.
+  /// \param parent is a parent acoording to countries.txt or cournties_migrate.txt.
+  /// \param downloadedChildren children partly or fully downloaded.
+  /// \param availChildren fully available children. No of its mwms have been downloaded.
+  /// \note. This method puts to localChildren only real maps which have been written in coutries.txt or cournties_migrate.txt.
```

Comment:
Fixed:

  /// \note. This method puts to |downloadedChildren| and |availChildren| only real maps (and its ancestors)
  /// which have been written in coutries.txt or cournties_migrate.txt.